src/warc2zim/templates/head_insert.html

```diff
@@ -0,0 +1,207 @@
+        ),
+        replace: "vimeo-cdn.fuzzy.replayweb.page/$1",
+      },
+      {
+        match: new RegExp(".*player.vimeo.com/(video/[\\d]+)\\?.*"),
+        replace: "vimeo.fuzzy.replayweb.page/$1",
+      },
+    ];
+
+    for (const rule of fuzzy_rules) {
+      const new_path = path.replace(rule.match, rule.replace);
+      if (new_path != path) {
+        return new_path;
+      }
+    }
+    return path;
+  };
+
+  wbinfo = {};
+  wbinfo.rewrite_function = function (url, useRel, mod, doc) {
+    if (!url) return url;
+
+    url = url.toString();
+
+    if (url.startsWith(wbinfo.wombat_host)) return url;
+
+    for (const prefix of [
+      "#",
+      "about:",
+      "data:",
+      "blob:",
+      "mailto:",
+      "javascript:",
+      "{",
+      "*",
+    ]) {
+      if (url.startsWith(prefix)) {
+        return url;
+      }
+    }
+
+    var absolute_url;
+    if (url.startsWith("//")) {
+      absolute_url = "{{ orig_scheme }}" + url;
+    } else if (url.startsWith("/")) {
+      // We have a absolute path without host.
+      // So it is a absolute path relative to the original host.
+      absolute_url = String(new URL(url, "{{ orig_url }}"));
+    } else {
+      // Relative path or full url.
+      // Let's build relative to our current url (`URL` will take care of relative vs full url)
+      absolute_url = String(new URL(url, window.location.href));
+    }
+
+    var entry_path;
+    if (absolute_url.startsWith(wbinfo.prefix)) {
+      // The absolute_url start with our prefix.
+      // It means that `url` was a relative or already processed url.
+      // We can simply remove our prefix to found our entry's path.
+      entry_path = absolute_url.substring(wbinfo.prefix.length);
+    } else {
+      // Remove potential scheme.
+      entry_path = absolute_url.replace(/^\w+:?\/\//i, "");
+    }
+
+    // Now we have a entry's path "as seen by the website".
+    // We need to reduce it to what we have stored in the zim file.
+    const reduced_path = reduce(entry_path);
+
+    var final_url = wbinfo.prefix + reduced_path;
+
+    final_url = new URL(final_url);
+    if (final_url.search) {
+      final_url.pathname =
+        final_url.pathname + encodeURIComponent(final_url.search);
+      final_url.search = "";
+    }
+
+    var encoded = String(final_url);
+
+    return encoded;
+  };
+
+  // Seems to be used only to send message to. We don't care ?
+  wbinfo.top_url = current_url;
+
+  // Seems to be used to generate url for blobUrl returned by SW.
+  // We don't care (?)
+  wbinfo.url = "{{ orig_url }}";
+
+  // Use to timestamp message send to top frame. Don't care
+  wbinfo.timestamp = "";
+
+  // Use to send message to top frame and in default rewrite url fonction. Don't care
+  wbinfo.request_ts = "";
+
+  // The url only which we are served.
+  wbinfo.prefix = current_url.substring(
+    0,
+    current_url.length - "{{ path }}".length,
```

Comment:
```suggestion
    current_url.length - ("{{ path }}".length),
```
better be safe with JS!